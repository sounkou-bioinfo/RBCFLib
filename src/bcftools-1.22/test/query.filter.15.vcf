##fileformat=VCFv4.2
##contig=<ID=chr1,length=5>
##INFO=<ID=TAG,Number=.,Type=String,Description="Some tag">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	1	.	*	*	.	.	TAG=a,b,c
chr1	2	.	*	*	.	.	TAG=a
chr1	3	.	*	*	.	.	TAG=.
chr1	4	.	*	*	.	.	TAG=.,.
chr1	5	.	*	*	.	.	TAG=a,.,c
