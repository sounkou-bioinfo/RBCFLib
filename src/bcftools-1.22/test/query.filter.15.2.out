a,b,c
a
a,.,c
